app/controllers/nonprofit_applications_controller.rb

```diff
@@ -0,0 +1,25 @@
+class NonprofitApplicationsController < ApplicationController
+  before_action :authenticate_nonprofit!
+
+  def new
+    @nonprofit_application = current_nonprofit.nonprofit_applications.build
+  end
+
+  def create
+    @nonprofit_application = current_nonprofit.nonprofit_applications.build nonprofit_application_params
+    if @nonprofit_application.save
+      SendNonprofitApplicationEmail.execute @nonprofit_application
+      redirect_to root_path, flash: { success: t("nonprofit_applications.create.success") }
```

Comment:
Hm maybe the root_path isn't the best place to redirect to? Might feel slightly better to direct them to the NPO apply page